testing/python_mldb_interface_test.py

```diff
@@ -0,0 +1,122 @@
+#
+# python_mldb_interface_test.py
+# Mich, 2016-01-25
+# This file is part of MLDB. Copyright 2015 Datacratic. All rights reserved.
+#
+import unittest
+
+if False:
+    mldb = None
+    mldb_wrapper = None
+
+mldb = mldb_wrapper.wrap(mldb)
```

Comment:
if `mldb_wrapper` is a global, then we don't need to call a method with an argument, right? We can just do something simpler like `mldb = api_v2()` and that will return a nice object?